package.json

```diff
@@ -34,6 +34,7 @@
     "cookie-parser": "^1.4.5",
     "express": "^4.17.1",
     "jsonwebtoken": "^8.5.1",
+    "jwt-decode": "^3.1.2",
```

Comment:
This has been added but I don't see any actual references to this. Am I missing something?

Reply:
I think I might have accidentally deleted this during the rebase. The code that uses this is in #30 .
I've added it again as it was missing.

Reply:
Yes, looking back through the commits, I must have deleted it somewhere or done something else to it but I've no idea where that error must have occurred.
This adds `jwt-decode` back in.